### ISSUE TITLE
v0.6 kfctl_k8s_istio.yaml config should point to 0.6 manifests repos 

### ISSUE THREAD

**Issue by abhi-g** [kubeflow/kubeflow#3862]
/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]
When I use kfctl_k8s_istio.yaml config for kfctl --config option, the yaml spec points to manifests from master.

**What did you expect to happen:**
I expect the config to point to the manifests from the v0.6-branch

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]
Possibly the documenation https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/ should also point the config file from the v0.6 branch rather than master?

**Environment:**

- Kubeflow version: (version number can be found at the bottom left corner of the Kubeflow dashboard): v0.6.1
- kfctl version: (use `kfctl version`): v0.6.1
- Kubernetes platform: (e.g. `minikube`) Existing k8s  or Minikube
- Kubernetes version: (use `kubectl version`): 1.15.x
- OS (e.g. from `/etc/os-release`): Linux

**Comment by yanniszark** [comment]
The same problem exists for all current configs. (GCP, AWS, existing_arrikto)

**Comment by yanniszark** [comment]
@abhi-g @lluunn I already had a PR ready for this, so I submitted it in #3864.
It fixes the issue for all config files.
Please take a look when you can :smile:

**Comment by abhi-g** [comment]
Thanks Yannis.

**Comment by yanniszark** [comment]
New PR: https://github.com/kubeflow/kubeflow/pull/3867

**Comment by yanniszark** [comment]
Also made a PR in website to fix the docs:
https://github.com/kubeflow/website/pull/1071

Once that is merged, we should be able to close this issue.

**Comment by jlewi** [comment]
Closing this as it should be fixed as part of the v0.6.2 release
https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/

The config now pins to specific commits of the manifests repo
https://github.com/kubeflow/kubeflow/blob/c54401e6c865360344e852ab801278f47d91f5e8/bootstrap/config/kfctl_k8s_istio.0.6.2.yaml#L11
